public/docs/_examples/dependency-injection/dart/web/heroes/hero.service.2.dart

```diff
@@ -0,0 +1,17 @@
+// #docregion
+import "package:angular2/core.dart";
+import "hero.dart";
+import "mock-heroes.dart";
+import "../logger.service.dart";
+
+@Injectable()
+class HeroService {
+  Logger _logger;
+  //#docregion ctor
+  HeroService(this._logger) {}
+  //#enddocregion ctor
+  getHeroes() {
```

Comment:
return type? There are several without. IMHO public API should have types in Dart.